documentation/src/main/asciidoc/topics/ref_jmx_xsite_components.adoc

```diff
@@ -7,7 +7,9 @@ The `org.infinispan:type=Cache` component provides the following JMX MBeans:
 
 * `XSiteAdmin` exposes cross-site operations that apply to specific cache instances.
 * `StateTransferManager` provides statistics for state transfer operations.
-* `InboundInvocationHandler` provides statistics and operations for asynchronous and synchronous cross-site requests.
+* `RpcManager` provides statistics for cross-site requests.
```

Comment:
Maybe it's the context but "requests" seems a little vague to me. I'm not sure but I feel like we should expand this a little more or s/requests/operations because "operations" is more general usage.

A second thing here is maybe we should always say "cross-site replication" or "cross-site backups" instead of "cross-site" on its own. It's a specific feature/component so should be consistent and clear that's what we're talking about.

Reply:
requests here mean the network messages, i.e. the number of requests sent (and response times) and the number of requests received. Not sure if there are a better word for that but I can use "operations" instead.

Reply:
Yeah, in that case "requests" does makes more sense than "operations". How about "provides statistics about network requests for cross-site replication."

Reply:
sounds good :)